<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnPause">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>‚è∏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnReturn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>‚èÆ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnBusca">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>üîé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnRepeat">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>‚ü≥</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSkip">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>‚è≠</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnContinue">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>‚ñ∂</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Txtbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListView" name="LVplaylist">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
